src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import core.basesyntax.service.Product;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class Storage {
+    private static Map<Product, Integer> productsMap = new TreeMap<>();
```

Comment:
why TreeMap?
let's use HashMap

Reply:
I wanted to store data in alphabetical order. But ok, I will replace it with HashMap